README.md

```diff
@@ -475,6 +475,9 @@ Other Style Guides
     // bad
     const name = "Capt. Janeway";
 
+    // bad
```

Comment:
Although, that's not strictly true. Sometimes you want to use them for newlines I think.